data/remnant/remnant 1 introduction.txt

```diff
@@ -2435,6 +2435,59 @@ mission "Remnant: Salvage 6"
+				`	"Yes, I can."`
+					goto agreement
+				`	"Sorry, not right now."`
+					defer
+			label agreement
+			`	"Thank you." They hand you the case. "Take it to <destination> by <date>. Torza will meet you there."`
+	on complete
+		event "remnant: scanning tolerance"
+		conversation
+			`You step out of the <ship> to see a non-descript Remnant standing attentively next to the landing pad. "Captain <first>! We have been waiting for you." They lead you into a nearby bunker, inside a single Remnant is standing over what appears to be a hologram display of fortifications and what appears to be some troops and ships moving around.`
+			`	"Torza, Captain <first> is here." The Remnant's gestures convey a deep sense of respect.`
+			`	The Remnant steps forward and accepts the case from you. The case pops open, and he slots the crystal into a spot on the display. He nods as the display pops up several confirmations of data integrity. "Well, you have certainly made many ripples since you arrived in our little corner of the galaxy." He gestures and the large display zooms in on a spot to show you walking through a spaceport outfitter. The display shifts to show you helping wounded onto your ship. The scene shifts again to show you launching the small stealth surveillance satellite. Then the table shifts back to the display of troop movements.`
+			`	"You have not been with us too long, but you have shown yourself to be trustworthy and eager to help." He gestures at the display. "As you can see, your actions have not gone unobserved. We have decided to amend your IFF registration as a result. He points to a set of slots next to the table similar to the ones that are installed on your ship. "Please place both your licenses in those slots, and I will confirm your side of the change."`
+			choice
+				`	(place your licenses in the designated spots.)`
+					goto comply
+				`	"You have been watching me?"`
+			`	"Yes. There is very little that goes on in our worlds that is not observed by someone, and sometimes it garners more attention than others." He looks at you. "You are one such. As a stranger among us, you were subject to much attention, although you probably did not notice most of it. Some felt it was too risky to trust you as far as we did, but the consensus was that you are worth the trust. Now," he gestures towards the console again. "place your licenses there please."`
+			label comply
+			`	He slots several things that look similar to but different from your licenses into slots on his side of the table and taps several controls. "You are not  yet one of us, but this is one step closer to being such. Your IFF signature has been updated, and our ship's databases will be updated within a day." He removes the licenses and hands yours back to you with a relieved expression. "We are a curious people, and we fully understand the search for knowledge and to learn more about one's surroundings. But we also know how important it is to maintain our secrets.`
```

Comment:
There is an extra space here, in `you are not  yet one of us,`

Reply:
Thanks! It is now fixed. Also found another double space elsewhere in the same file. lol.